src/components/ui/CopyLink/CopyLink.tsx

```diff
@@ -0,0 +1,75 @@
+import { AnimatePresence, motion } from 'framer-motion';
+import { useEffect, useState } from 'react';
+
+export const CopyLink = ({ link }: { link: string }) => {
+  const [copied, setCopied] = useState(false);
+
+  // Reset copied state after 1.5 seconds to allow user to copy again
+  useEffect(() => {
+    if (copied) {
+      const timer = setTimeout(() => {
+        setCopied(false);
+      }, 1500);
+      return () => clearTimeout(timer);
+    }
+  }, [copied]);
+
+  const copyLink = () => {
+    navigator.clipboard.writeText(link);
+    setCopied(true);
+  };
+
+  return (
+    <div className="flex items-center gap-[10px] h-10 rounded-full justify-between px-2 pr-4 shadow-[0_0_8px_rgba(4,41,49,0.15)] text-[13px] relative">
```

Comment:
Tailwind already has a 10px gap! It's under the name 'gap-2.5'. Check out the documentation right here: https://tailwindcss.com/docs/gap.

----

Whenever the value is not in Tailwind, as for the 13px, we always try to keep it in `rem` unit of measure. To transform it from `px` to `rem` I'd recommend you to use https://nekocalc.com/px-to-rem-converter as it's quite simple to use.

Whenever you can't find a specific value in Tailwind, like 13px for instance, we always try to convert it into the rem unit. A tool I'd recommend is https://nekocalc.com/px-to-rem-converter. It's quite easy to use.

(Please look throughout the file as this is present in different spots!)

---

Looking at the link, it seems a bit close to the left side. I reckon the padding in the Figma design was set to 16px, not 8px.

Reply:
I think you're right. I mistakenly changed it back to 8px because the right end padding has to be 8px. So I thought it would be consistent.